nvflare/private/fed/server/fed_server.py

```diff
@@ -336,12 +336,13 @@ def _listen_command(self, request: Message) -> Message:
             return make_cellnet_reply(F3ReturnCode.OK, "", fobs.dumps(return_data))
         elif command == ServerCommandNames.UPDATE_RUN_STATUS:
             execution_error = data.get("execution_error")
-            if execution_error:
-                with self.lock:
-                    run_process_info = self.engine.run_processes.get(job_id)
+            with self.lock:
```

Comment:
This `ServerCommandNames.UPDATE_RUN_STATUS` will only be called when a job is finished?

Reply:
Yes